src/features/metadata-instance-editor/Instance.js

```diff
@@ -590,8 +581,7 @@ class Instance extends React.PureComponent<Props, State> {
             shouldShowCascadeOptions,
             isCascadingOverwritten,
         }: State = this.state;
-        const isProperties = template.templateKey === TEMPLATE_CUSTOM_PROPERTIES;
-        const isEditing = this.isEditing();
```

Comment:
I would imagine the performance impact is minor but consider that you are now calling `isEditing()` n times instead of storing the result

Reply:
agree, I think the original is cleaner too